tally_ho/apps/tally/views/corrections.py

```diff
@@ -193,6 +196,14 @@ def save_recon(post_data, user, result_form):
                     value = False
                 corrections[v1.name] = value
 
+            # Error occured at data entry 1 stage
+            if int(value) != v1.data:
+                data_entry_1_errors += 1
+
+            # Error occured at data entry 2 stage
+            if int(value) != v2.data:
+                data_entry_2_errors += 1
+
     if len(corrections) < mismatched:
         raise ValidationError(
             _(u"Please select correct results for all mis-matched votes."))
```

Comment:
Do we need the `u` here?

Reply:
Not it's actually, will remove any occurrence of this